packages/@storefront/flux-capacitor/test/unit/core/requests/utils.ts

```diff
@@ -572,12 +572,24 @@ suite('requests helpers', ({ expect, stub, spy }) => {
       )).to.eql({ a: 'b', c: 'd', e: 'f' });
     });
 
-    it('should merge source if tranformation returned falsey', () => {
+    it('should merge source if transformation returned falsey', () => {
       expect(RequestHelpers.chain(
         utils.normalizeToFunction(<any>{ a: 'b' }),
         (x) => null,
         utils.normalizeToFunction({ e: 'f' })
       )).to.eql({ a: 'b', e: 'f' });
     });
   });
+
+  describe('attachSessionId()', () => {
+    it('should return a function that is called with the given request and add sessionId to the request', () => {
```

Comment:
Simplify to something like: "it should return a function which applies the session ID to the request".